server.js

```diff
@@ -46,13 +47,20 @@ app.get('/favicon.ico', (req, res) => {
 
 app.all('/graphql', (req, res) => res.redirect('/'));
 
-app.use('/', cors(), morgan('combined'), graphqlHTTP(() => {
+app.use('/', cors(), morgan('combined'), graphqlHTTP(request => {
+  info('----------');
+
   loaders.clearAll();
 
+  const accessToken = request.headers['x-access-token'];
```

Comment:
I've been using https://github.com/skevy/graphiql-app which will let you set headers (but is pretty buggy). Force is already sending this header.